modules/local/uniref90_rhea_filter/main.nf

```diff
@@ -1,6 +1,6 @@
 process UNIREF90_RHEA_FILTER {
     label 'process_single'
-    container 'quay.io/biocontainers/pyfastx:2.2.0--py39h0699b22_0'
+    container 'quay.io/biocontainers/biopython:1.78'
```

Comment:
no need to change this at all, but there are more recent versions of biopython (mgnify-pipelines-toolkit uses 1.82 for example), just in case we want to note that for the future